src/core/db/repository/base.py

```diff
@@ -64,6 +64,10 @@ async def create_all(self, objects: list[DatabaseModel]) -> None:
         """Создает несколько объектов модели в базе данных."""
         self._session.add_all(objects)
 
+    async def count_all(self, instance: DatabaseModel) -> int:
+        """Возвращает количество юнитов категории."""
+        return await self._session.scalars(select(func.count()).select_from(instance))

```

Comment:
`instance` убери, он тут не нужен. 
```suggestion
        return await self._session.scalars(select(func.count()).select_from(self._model))

```